### PR TITLE
refactor(progress-bar-expressive): rename expressive-loader to progress-bar-expressive

### LINKED ISSUE #2289: progress-bar-expressive: rename expressive-loader to progress-bar-expressive
### Description

Rename `expressive-loader` module to `progress-bar-expressive` for more semantic naming and consistency with our other progress modules.

### PR DESCRIPTION
**Pull request by cordeliadillon**
Fixes #2289 

- [ ] This PR contains CSS changes
- [x] This PR does not contain CSS changes

## Description
Rename `expressive-loader` module to `progress-bar-expressive` module for more semantic naming and consistency with other progress modules. Update CSS class names, CSS variables, helper JS, and docs accordingly.

## Checklist

- [x] I verify the build is in a non-broken state
- [x] I verify all changes are within scope of the linked issue

### CODE REVIEW

File: dist/progress-bar-expressive/progress-bar-expressive.css [hunk 1]
```diff
@@ -0,0 +1,277 @@
+:root {
```
Comment (cordeliadillon) [inline dist/progress-bar-expressive/progress-bar-expressive.css:1]: Only changes in this file are `expressive-loader` --> `progress-bar-expressive`

File: src/less/progress-bar-expressive/stories/progress-bar-expressive.stories.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+export default { title: "Skin/Progress Bar Expressive" };
```
Comment (cordeliadillon) [inline src/less/progress-bar-expressive/stories/progress-bar-expressive.stories.js:1]: Changed storybook name and CSS classes.

**Review by ArtBlue — APPROVED** [review]
Looks good.

**Review by ianmcburnie — APPROVED** [review]
LGTM. We forgot about `progress-bar-marquee`. LOL.

**Pull request merged**